### ISSUE TITLE
Cannot load form in #30

### ISSUE THREAD

**Issue by mixmixmix** [road86/bahis-desk#34]
Blocking issue for #30 

user: paba, ghatail

After sucesfully logging in (through problems in https://github.com/road86/bahis-desk/issues/33 ) and syncing, when trying to load form `Configuration->Staff->New Submission`, I get "Something went wrong" message printed and following that
```
TypeError: Cannot read property 'division' of undefined
Form._this.getUserInput
src/components/Form/index.tsx:91
  88 |   for (let prop of userInputProperties) {
  89 |     userInput = {
  90 |       ...userInput,
> 91 |       [prop[0].slice(1)]: userLocationInfo[prop[1].split('@')[1]]
     | ^  92 |     }
  93 |   }
  94 | }
View compiled
Form.render
src/components/Form/index.tsx:150
  147 |       value: 'Bangla',
  148 |     },
  149 |   ],
> 150 |   userInputJson: this.getUserInput(dataJson),
      | ^  151 | };
  152 | 
  153 | // const goBack = () => this.props.history.goBack();
View compiled
▶ 19 stack frames were collapsed.
Form._callee$
src/components/Form/index.tsx:77
  74 |   const userInfo = await ipcRenderer.sendSync('fetch-userlist');
  75 |   if (formDefinitionObj != null) {
  76 |     const { definition, formChoices } = formDefinitionObj;
> 77 |     this.setState({ formDefinition: definition, formChoices, userLocationInfo, userInfo });
     | ^  78 |   }
  79 | }
  80 | 
  ```

**Comment by mixmixmix** [comment]
It is *not* fixed. For `paba` and `ghatail` still the same error. However the form renders correctly for `rupsa`, with one caveat.

**Comment by ChasNelson1990** [comment]
This was actually a problem with my new bahis-serve API endpoint... turns out (and this could be what makes the difference between RUPSA and GHATAIL) the `core.branch_catchment_area` table maps GHATAIL `branch_id` to GHATAIL `geo_id` but maps RUPSA `branch_id` to KHULNA `geo_id`, i.e. the root division not the upazilla itself. Fixed in a commit to road86/bahis-serve#9 - this issue is closed and GHATAIL now replicates #35
